qrcodereaderview/src/main/java/com/dlazaro66/qrcodereaderview/QRCodeReaderView.java

```diff
@@ -231,11 +245,11 @@ public void setFrontCamera() {
   // Called when camera take a frame
   @Override public void onPreviewFrame(byte[] data, Camera camera) {
     if (!mQrDecodingEnabled || (decodeFrameTask != null
-        && decodeFrameTask.getStatus() == AsyncTask.Status.RUNNING)) {
+            && decodeFrameTask.getStatus() == AsyncTask.Status.RUNNING)) {
```

Comment:
I'd prefer to keep the current codestyle. I know is not the best, but I don't want to change it, since I'm planning to adopt the Square codestyle (and change the project style at once) and add it as a constraint to propose a PR